vignettes/programming_strategy.Rmd

```diff
@@ -679,9 +710,22 @@ arguments which are not passed as `exprs()` argument (e.g. `new_var = VAR1` or
 ### END DEPRECATION
 ```
 
+**Phase 3:**  If an argument is removed and is not replaced, an **error** must be generated:
+
+```
+### BEGIN DEPRECATION
+  if (!missing(old_param)) {
+    deprecate_stop("x.y.z", "fun_xxx(old_param = )", "fun_xxx(new_param = )")
+  }
+### END DEPRECATION
+```
+
+
+
 ## Unit Testing
 
-Unit tests for deprecated functions and arguments must be added to the test file [^1] of the function to ensure that a warning or error is issued.
+Unit tests for deprecated functions and arguments must be added to the test 
+file [^1] of the function to ensure that a warning or error is issued.
 
 [^1]: For example, if `derive_var_example()` is going to be deprecated and
 it is defined in `examples.R`, the unit tests are in
```

Comment:
Should we update line 735 to add the class issued by `deprecate_inform()`?

Reply:
yes!  ACtually I was wondering if we should create `deprecate_inform()` in this PR and start working on `derive_var_dthcaus()`  or agree and finish the strategy first and then work on the function in a separate PR?

Reply:
I would implement `deprecate_inform()` in this PR. Otherwise, we are referring to a function which doesn't exist.